Add aggregation from dns

TheTedLab opened the pull request:
Add selenium aggregation from dns-shop.ru
Upgrade WebDriverWait to wait until visibility of element Add offers files
Update requirements.txt and .gitignore

TheTedLab committed:
Add aggregation from dns

Add selenium aggregation from dns-shop.ru
Upgrade WebDriverWait to wait until visibility of element
Add offers files
Update requirements.txt and .gitignore

Izentall commented on src/server/aggregator/data-aggregator.py:
```diff
@@ -313,13 +323,135 @@ def get_data_citilink(date):
+
+    block = driver.find_element(By.CLASS_NAME, 'products-list__content')
+    for item_id in range(1, page_limit + 1):
+        item = block.find_element(By.CSS_SELECTOR,
```
Может стоит сюда (424 строка) проверку вставить, что цена присутствует на странице? Выполнять find_element в цикле, выходить из него по условию наличия цены и по максимуму итераций, например, 10

TheTedLab replied:
Исправил

TheTedLab committed:
Add NoSuchElementException error handler

Add try-except block for NoSuchElementException with cycle for 10 attempts
Add offers files
